src/doc/src/reference/workspaces.md

```diff
@@ -15,6 +15,9 @@ The key points of workspaces are:
   sections in `Cargo.toml` are only recognized in the *root* manifest, and
   ignored in member crates' manifests.
 
+The *workspace root* is the directory where the workspace's `Cargo.toml` 
+is located.
```

Comment:
This feels a bit disjointed to define this term here and in this way.

Reply:
Sorry this is a bit of my own immature consideration.

[cargo book](https://doc.rust-lang.org/stable/cargo/reference/workspaces.html#workspaces):

> All packages share a common [Cargo.lock](https://doc.rust-lang.org/stable/cargo/guide/cargo-toml-vs-cargo-lock.html) file which resides in the workspace root.
> All packages share a common [output directory](https://doc.rust-lang.org/stable/cargo/guide/build-cache.html), which defaults to a directory named target in the workspace root.

The use of `workspace root` has already appeared in previous descriptions, and I personally feel that it may be easier to read the definitions by placing them in the nearest location.

Reply:
I agree with epage. This paragraph is more like an overview. Putting term definition here seems odd. This might be more appropriate if moved to "The `[workspace]` section" before "Root package", such like

> The directory where the workspace's `Cargo.toml` is located then becomes so-called the _workspace root_.

Reply:
Done. Thank you.